tsconfig.json

```diff
@@ -8,5 +8,6 @@
     "resolveJsonModule": true,
     "skipLibCheck": true
   },
-  "include": ["**/*.ts"]
+  "include": ["**/*.ts"],
+  "exclude": ["cases/paseri/src/*.ts"]
```

Comment:
Things getting complicated here @moltar.

Would be nice to have some kind of isolation for all the libraries :grin:. I have the feeling that sooner or later we'll have some problems due to having all those libraries in a single npm project.